lib/fastlane/plugin/mango/helper/docker_commander.rb

```diff
@@ -31,7 +30,7 @@ def self.delete_container(container_name:)
         Actions.sh("docker rm #{container_name}") if container_name
       end
 
-      def self.disconnect_network_bridge(container_name: container_name)
```

Comment:
I believe that this class var was accessible only in the context of `DockerHelper`. I checked and we define this variable in all the calls, so it is safe to just remove and no other fixes are needed.